swagger/src/test/scala/org/scalatra/swagger/SwaggerSpec.scala

```diff
@@ -420,7 +421,7 @@ class SwaggerSpec2 extends ScalatraSpec with JsonMatchers {
     val JObject(definitions) = j
     definitions.map {
       case (name, definition) =>
-        verifyFields(definition, swaggerJsonJValue \ "securityDefinitions" \ name, "type", "name", "in", "flow", "authorizationUrl", "scopes")
+        verifyFields(definition, swaggerJsonJValue \ "securityDefinitions" \ name, "type", "name", "description", "in", "flow", "authorizationUrl", "scopes")
```

Comment:
Oh really! So you don't need to add "description" field in `SwaggerSerializers.scala`, right?